src/charts/bar.js

```diff
@@ -761,19 +761,19 @@ define(function(require) {
         };
 
         /**
-         * Gets or Sets the hasSingleHover status.
+         * Gets or Sets the hasSingleBarHighlight status.
          * If the value is true (default), the only the hovered bar
          * will switch to darker color. If the value is false, only the hovered bar will stay
          * the same while the rest of the bars will switch to darker colors.
-         * @param  {boolean} _x     Should darken the hovered bar
-         * @return { boolean | module} Is hasSingleHover used or Chart module to chain calls
+         * @param  {boolean} _x        Should darken the hovered bar
```

Comment:
Nit: I would say 'highlight' instead of 'darken', in preparation for the next change